src/app/modules/main/node_section/controller_interface.nim

```diff
@@ -0,0 +1,36 @@
+import json
+import status/types/[rpc_response]
```

Comment:
@borismelnik new refactored part should not rely on the old code from `status-lib`.

Reply:
got it